web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/dialog/DialogController.kt

```diff
@@ -86,9 +87,18 @@ class DialogController @Inject constructor(
                 .check(Policies.tilgangTilBruker.with(fnr))
                 .get(Audit.describe(CREATE, Person.Henvendelse.Les, AuditIdentifier.FNR to fnr)) {
                     val context = lagSendHenvendelseContext(fnr, request)
-
-                    henvendelseUtsendingService.sendHenvendelse(lagSporsmal(sporsmalsRequest, sporsmalsRequest.sak.temaKode, context), Optional.empty(), Optional.of(sporsmalsRequest.sak), context.enhet)
-                    Response.ok().build()
+                    val henvendelseStatus = henvendelseUtsendingService.sendHenvendelse(
+                            lagSporsmal(sporsmalsRequest, sporsmalsRequest.sak.temaKode, context),
+                            Optional.empty(),
+                            Optional.of(sporsmalsRequest.sak),
+                            context.enhet);
+                    when (henvendelseStatus.status) {
```

Comment:
Syns vi burde titt litt på hvordan dette generaliserer. 
For tror ikke dette er eneste stedet hvor bedre feilmeldinger kunne være nyttig for oss.